fstest/fstests/fstests.go

```diff
@@ -701,11 +702,15 @@ func Run(t *testing.T, opt *Opt) {
 				{"trailing dot", "trailing dot."},
 				{"invalid UTF-8", "invalid utf-8\xfe"},
 				{"URL encoding", "test%46.txt"},
+				{"many chars", "quite-long-file-name-with-more-than-fifty-chars.txt"},
```

Comment:
Does this make the tests pass?

I note these are failing in the integration tests at the moment. (I'm sure they were passing when we merged linkbox so something has changed!)

Reply:
This caused an error for Linkbox before the fix, but worked after the fix. However, it is better to move this into a separate commit, as it affects the tests of other remotes.